dev-cpp/gsl/metadata.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+<maintainer type="person">
+	<email>gentoo@taujhe.de</email>
+	<description>Jan Henke</description>
```

Comment:
Since you're not a gentoo developer, you need to add proxy maint after yourself:
```
        <maintainer type="project">
                <email>proxy-maint@gentoo.org</email>
                <name>Proxy Maintainers</name>
        </maintainer>
```